### ISSUE TITLE
New to ESP32

### ISSUE THREAD

**Issue by JRM-PT** [DerianAndre/Smart-Watch-Winder#1]
Hi,

I'm new to this kind of programmable electronic. I've setup a Raspberry Pi Music Centre but following a step by step guide.
I liked your project and I might do a similar one but I got some doubts:
You state a LN293D on your parts list but the DC motor driver reference is L293D. Have you measured the actual voltage on the motor when running?
I also bought a cheap AliExpress watch winder and got a geared DC motor but the controls suck and one can't regulate Turns Per Day, speed or start/stop timing.
How do you control this things in your code? I couldn't find it.
Finally, I'd use a few tips regarding the installation of the code itself: is it  just burn a micro SD card?
Thanks a lot

**Comment by DerianAndre** [comment]
Hi,
I didn't measued the voltage.
I haven't touched this project for a long time but its not a lot of code at least for the ESP32. What you can do is trying to calculate TPD with the voltage you provide and the time it takes to do a turn and just use timing (wont be accurate but it will work fine)-
For running the code, this was designed to use an ESP32 so it must be done using arduino or platformio with vscode

**Comment by JRM-PT** [comment]
Thanks for your answer. I know that my geared DC motor gives about 10 turns per minute. I think I will take the small risk and buy an ESP32 and L293 to make this project.

João Moreira
________________________________

**Comment by DerianAndre** [comment]
You wont regret using an ESP32. Trust me. Hope this project serves a base for yours :)

**Comment by JRM-PT** [comment]
Yes it will my base of work. First of all I have to understand the ESP32 and your code. No worries, something to do on a relaxed way.

João Moreira
________________________________
